src/AppInstallerCLICore/Workflows/OpenUrlWorkflow.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <AppInstallerProgress.h>
+#include "ExecutionContext.h"
+
+#include <filesystem>
+#include <optional>
+
+// ShellExecuteInstallerHandler handles installers run through ShellExecute.
+// Exe, Wix, Nullsoft, Msi and Inno should be handled by this installer handler.
+namespace AppInstaller::CLI::Workflow
+{
+    // Install is done through invoking ShellExecute on downloaded installer.
+    // Required Args: None
+    // Inputs: Manifest?, InstallerPath, InstallerArgs
+    // Outputs: None
+    void OpenUrlInDefaultBrowser(Execution::Context& context);
```

Comment:
>void OpenUrlInDefaultBrowser(Execution::Context& context); [](start = 4, length = 58)

The workflow organization is not super crisp.  I would have put this all in ShowFlow, as it is specific to show right now.